### DISCUSSION TITLE
Disable auto-update of entity state when sliding

### DISCUSSION THREAD

**Discussion by KlumzyKlein** [Clooos/Bubble-Card#686] [Feature requests]
Is there any way to disable a light entity from updating until I have finished swiping?

Currently, when I drag a light slider, the haptic feedback continuously vibrates and the light updates multiple times during the action.

**Comment by marlind89** [comment]
+1 would like this as well :) 

The default behavior when sliding in home assistant doesn't update until swiping is finished so I think it would make sense to be able to configure this.

![image](https://github.com/user-attachments/assets/c7f0b8aa-b13e-4408-bf33-a00313e43613)

**Comment by willliamchan** [comment]
Second this. The real-life scenario is I've got two Daikin air conditioning units and I don't dare to use the slider as Daikin limits the API call that can be made per min / per hour / per day.
